.github/workflows/build-preview-environment.yml

```
@@ -0,0 +1,108 @@
+      tag: ${{ needs.deploy-dev-pr-environment.outputs.env_name }}
+
+  deploy-preview:
+    needs: [deploy-dev-pr-environment,build-wf-service,build-backoffice,build-kyb,build-dashboard]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger workflow in another repo
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const refValue = ('${{ github.event_name }}' === 'workflow_dispatch') 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)